### ISSUE TITLE
Training LatentInpaintDiffusion model on my datasets, but the results are poor, how can i solve it?

### ISSUE THREAD

**Issue by LM7XAw** [Stability-AI/stablediffusion#419]
Thank you for your great work on this model! I'm training it on my own defect dataset, but the results are not coming out well. What might be the cause of this? How can I improve it?

I'm using the configuration file `configs/stable-diffusion/v2-inpainting-inference.yaml` with the SD 2.0-inpainting checkpoint as the pre-trained model, but the output is still not satisfactory.

input_gs:
![ad7518b474573fffa99f9d0df3c4395c](https://github.com/user-attachments/assets/1fd46f2c-fb94-42d5-aa8e-4cabb4315075)

masked_image:
![f44fff233568cb3a295621f9b03ec421](https://github.com/user-attachments/assets/5532917b-9dcd-45d6-bcce-bd65e392d5c2)

sample_gs:
![c7ad9dec1a330b86d05c0ace67a8af14](https://github.com/user-attachments/assets/48921a94-7e88-42a7-99f0-9003317a16af)

these images are scaled and padded to $512\times512$, and I randomly mask either the defect areas, the entire object, or a random region in the image.

During training, the loss initially decreases (see sample_gs), but then it starts increases again, and the generated images end up as pure noise. 

I'm training on two 3090 with a batch size of 2. I've tried changing the area of the defect mask, prompt, and learning rate, but the results are still not very satisfactory. Any tips for improving the results?
